### ISSUE TITLE
Project name in URL may not be encoded properly

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-maven-plugin#13]
See https://github.com/DependencyTrack/dependency-track/issues/329 for a description of the issue.

From stack trace in the issue, it looks like the spaces are not properly encoded in the URI.

**Comment by goneall** [comment]
A workaround to this issue is to specify the spdxDocumentNamespace in the spdx maven plugin configuration.  For example:

```
<plugin>
                <groupId>org.spdx</groupId>
                <artifactId>spdx-maven-plugin</artifactId>
                <executions>
                    <execution>
                        <id>build-spdx</id>
                        <phase>prepare-package</phase>
                        <goals>
                            <goal>createSPDX</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <spdxDocumentNamespace>http://spdx.org/documents/spdx-toolsv2.0-rc1</spdxDocumentNamespace>
...
```

**Comment by goneall** [comment]
@udaykor @stevespringett See the above work-around.  I'll work on a fix over the next week along with a few other fixes and push out a release.

I'll maintain a branch fixuriissue with the work in progress.

**Comment by goneall** [comment]
Resolved with PR #14
